### ISSUE TITLE
Generic Controller

### ISSUE THREAD

**Issue by DavisDev** [xerpi/ds3vita#1]
I testing, I discovered that it does not work with a generic Controller.
Maybe it has something to do with the vid and pid, even though I'm not sure, xD

**Comment by lazydevyo** [comment]
Yeah i have an non original ps3 controller and it doesn't work either, i hope theres gonna be support someday but i doupt
